config/routes.rb

```diff
@@ -14,6 +14,11 @@
   get("/auth/developer/sign-out", to: "sessions#signout")
   post("/auth/developer/callback", to: "sessions#callback")
 
+  resources :service_updates,
+            only: %i[index],
+            path: "service-updates",
```

Comment:
Is this the pattern we will want to use for all paths (ie, hyphen instead of snake case?). If so, we should change it at the top of the app in an initialiser (`ActionDispatch`) rather than for each individual route.